src/ResponseTagger.php

```diff
@@ -89,4 +90,22 @@ public function addTags(array $tags)
 
         return $this;
     }
+
+    /**
+     * Set tags on a response
+     *
+     * @param ResponseInterface $response Original response
+     * @param bool              $replace  Whether to replace the current tags
+     *                                    on the response
+     *
+     * @return ResponseInterface          Tagged response
+     */
+    public function tagResponse(ResponseInterface $response, $replace = false)
```

Comment:
Actually, I prefer to keep the method name `tagResponse()`. As the library is now built around PSR-7, we can assume that we’re talking about a PSR-7 response here. 

In the bundle, then, we may want to rename its `tagResponse(SymfonyResponse)` method to `tagSymfonyResponse()`, but that requires no changes here.

@dbu If you agree, this PR is ready to merge right away.